packages/react-composites/tests/browser/chat/hermetic/ImageGallery.test.ts

```diff
@@ -24,18 +24,22 @@ test.describe('ImageGallery Modal tests', () => {
     page,
     serverUrl
   }) => {
-    await page.goto(
-      buildUrlForChatAppUsingFakeAdapter(serverUrl, {
-        localParticipant: TEST_PARTICIPANTS[1],
-        remoteParticipants: [TEST_PARTICIPANTS[0], TEST_PARTICIPANTS[2]],
-        localParticipantPosition: 1,
-        sendRemoteInlineImageMessage: true,
-        inlineImageUrl: 'images/inlineImage-broken.png'
-      })
-    );
-
-    await page.locator(dataUiId('SomeImageId1')).click();
-    expect(await stableScreenshot(page)).toMatchSnapshot(`inline-image-gallery-modal-broken-link.png`);
+    try {
+      await page.goto(
+        buildUrlForChatAppUsingFakeAdapter(serverUrl, {
+          localParticipant: TEST_PARTICIPANTS[1],
+          remoteParticipants: [TEST_PARTICIPANTS[0], TEST_PARTICIPANTS[2]],
+          localParticipantPosition: 1,
+          sendRemoteInlineImageMessage: true,
+          inlineImageUrl: 'images/inlineImage-broken.png'
+        })
+      );
+    } catch (error) {
+      // This is expected to fail because the image url is a broken link,
+      // we want to swallow this error to prevent it from logging to the console
+      await page.locator(dataUiId('SomeImageId1')).click();
```

Comment:
Should it be in finally block instead, so we won't loose a screenshot if there is no errors?

Reply:
Moving to the finally block does cause the same logging issue.